config/bindings.php

```diff
@@ -92,6 +92,13 @@
     Interfaces\Remitter\RemitterVerification\AutomatedRemitterVerificationInterface::class => Entities\Remitter\RemitterVerification\AutomatedRemitterVerification::class,
 
     Interfaces\MerchantAccount\MerchantAccountInterface::class => Entities\MerchantAccount\MerchantAccount::class,
+    Interfaces\MerchantAccount\Transactions\MerchantAccountPaymentInterface::class => Entities\MerchantAccount\Transactions\MerchantAccountPayment::class,
```

Comment:
Can we also bind MerchantAccountTransactionRetrievedInterface to the base MerchantAccountTransactionRetrieved class? If a new type is added on the api, and an equivalent implementation is not found, the client will simply return the base implementation (without breaking).

Reply:
Done